lib/elixir_analyzer/exercise_test/two_fer.ex

```diff
@@ -66,11 +61,6 @@ defmodule ElixirAnalyzer.ExerciseTest.TwoFer do
         _ignore
       end
     end
-
-    # function one-liner
-    form do
-      def two_fer(_ignore \\ "you") when _ignore, do: _ignore
-    end
```

Comment:
It probably can, it would be a compile-time error since the `forms` from the test file are read at that time.  But could do some sort of check to see if the same form already exists.

Reply:
I created an issue for this: https://github.com/exercism/elixir-analyzer/issues/25